### ISSUE TITLE
[Accessibility] Carousel indicators not accessible by keyboard interaction

### ISSUE THREAD

**Issue by gselderslaghs** [materializecss/materialize#546]
### Before submitting...

- [X] I have searched for [duplicate or closed issues](https://github.com/materializecss/materialize/issues?utf8=%E2%9C%93&q=is%3Aissue).
- [X] I have read the [CONTRIBUTING document](https://github.com/materializecss/materialize/blob/master/CONTRIBUTING.md) and my issue is following the guidelines.
- [X] I have read the template completely before filling it in.

### Context

When using the Carousel component with indicators, the indicators are not accessible by keyboard interaction which causes reduced accessibility towards people who rely on keyboard input

### Current Behavior

Indicators not accessible by keyboard interaction

### Expected behavior

Reveal button should be accessible by keyboard interaction

### Possible Solutions or Causes

Implement the necessary functional attributes on initialization in order to make options accessible by keyboard interaction

### Your Environment

* Version used: 2.1.1

**Comment by gselderslaghs** [comment]
Addressed this issue in pull-request #547

**Comment by gselderslaghs** [comment]
Closing this as fixed in v2.2.1
